pkg/services/store/entity/sqlstash/sql_storage_server.go

```diff
@@ -77,6 +72,27 @@ func (s *sqlEntityServer) Init() error {
 
 	s.sess = sess
 	s.dialect = migrator.NewDialect(engine.DriverName())
+
+	// initialize snowflake generator
+	s.snowflake, err = snowflake.NewNode(rand.Int63n(1024))
+	if err != nil {
+		return err
+	}
+
+	// set up the broadcaster
+	s.broadcaster = &Broadcaster[*entity.Entity]{}
+	err = s.broadcaster.Start(func() (chan *entity.Entity, error) {
+		stream := make(chan *entity.Entity)
```

Comment:
Should we make this buffered?

Reply:
wouldn't hurt, right now it'd just rate-limit the poller to the throughput of the broadcaster afaict